doc/api/perf_hooks.md

```diff
@@ -165,6 +165,30 @@ obs.observe({ entryTypes: ['function'] });
 wrapped();
 ```
 
+### `performance.eventLoopUtilization([util])`
+<!-- YAML
+added:
+-->
+
+* `util` {Object} The result of a previous call to `eventLoopUtilization()`
+* Returns {Object}
+  * `idle` {number}
+  * `active` {number}
+  * `utilization` {number}
+
+The `eventLoopUtilization()` method returns an object that contains the
+cumulative duration of time the event loop has been both idle and active as a
+high resolution milliseconds timer. The event loop utilization as a percentage
+is calculated in the `utilization` value.
+
+`util` is an optional parameter that must be the result of a previous
+`eventLoopUtilization()` call. The `utilization` value is a calculation of the
+diff from the time when the previous call was made. Not from the total
+cumulative values.
+
+Passing in a user-defined object instead of the result of a previous call to
+`eventLoopUtilization()` will lead to undefined behavior.
```

Comment:
I'd maybe be a bit more specific about what "undefined behaviour" means here.

Reply:
I copied this from the `process.hrtim()` docs, but you're right. Better explanation would be good.